LDMP/calculate.py

```diff
@@ -522,7 +523,52 @@ def accept(self):
         pass
 
     def settings_btn_clicked(self):
-        self.iface.showOptionsDialog(currentPage=OPTIONS_TITLE)
+        self.iface.showOptionsDialog(self.iface.mainWindow(), currentPage=OPTIONS_TITLE)
+
+    def _validate_crs_multi_layer(self, layer_defn: list) -> bool:
```

Comment:
It would be worthwhile to offer users an option to automatically reproject any layers that are in a CRS other than that chosen in their user settings. Otherwise there is no easy way for a user to user layers they may have produced in a different CRS (within Trends.Earth that is), without re-running the analysis. Of course there might be boundary effects, due to the reprojection, there might not be transformations among the CRS if they were user defined, etc., but worth offering it as an option if possible.